src/plugin/renderer/render_pipeline_select_command.cpp

```diff
@@ -46,6 +51,35 @@ MStatus wmr::RenderPipelineSelectCommand::doIt(const MArgList& args)
 	}
 	else
 	{
+		// Convert to std::string
+		std::string path = skybox_path.asChar();
+
+		// Find the extension
+		auto found = path.find_last_of('.');
+		auto extension = path.substr(found + 1);
+
+		// File must have a valid texture extension
+		if (extension == "png" ||
+			extension == "jpg" ||
+			extension == "jpeg" ||
+			extension == "bmp" ||
+			extension == "dds" ||
+			extension == "hdr" ||
+			extension == "tga")
+		{
```

Comment:
Instead of checking on a hard-coded list, try-catch it (since WispRenderer throws) and log an error with LOGE.